### PR TITLE
A new quote for your appraisal.

### PR DESCRIPTION
**Pull request by AlvaroIsrael**
feat(👑): Add Marshall notable quote from tv show How I Met Your Mother.
chore(👔): Update recipe.md with TypeScript language.

### CODE REVIEW

**Review by AlvaroIsrael — COMMENTED** [review]
Approved.

**Review by keystove — CHANGES_REQUESTED** [review]
What's up with the quote? 🤔

File: README.md [hunk 1]
```diff
@@ -6,11 +6,15 @@
 **If you fork this then you can say you forked a pineapple.**
 
 
-## A quote
+## Quotes
 > "A pinnapple is a fruit. It is mostly yellow. If you fork u *gud*."
 
 ~ keystove (2021)
 
+> "...And a pineapple. Am I the only one who's curious about the pineapple?!"
```
Comment (AlvaroIsrael) [inline README.md:14]: @keystove This is the quote! It's a notable quote about a pineapple, you have to watch the episode to get it, but it's really funny!

**Review by AlvaroIsrael — COMMENTED** [review]
It's a notable quote about a pineapple, you have to watch the episode to get it, but it's really funny!

**Comment by keystove** [comment]
Hmm. Ok, I'll approve and merge it.

**Review by keystove — APPROVED** [review]

**Pull request merged**